src/commands/consent/upload-cookies-from-csv/impl.ts

```diff
@@ -13,6 +14,8 @@ export async function uploadCookiesFromCsv(
   this: LocalContext,
   { auth, trackerStatus, file, transcendUrl }: UploadCookiesFromCsvCommandFlags,
 ): Promise<void> {
+  doneInputValidation(this.process.exit);
```

Comment:
this is useful for testing, which confirms that example commands pass input validation